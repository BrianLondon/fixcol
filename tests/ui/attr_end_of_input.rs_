use fixed_derive::{ReadFixed};

#[derive(ReadFixed)]
struct Thing {
    #[fixed(width = 5, align =)]
    field: String,
}

pub fn main() {}
