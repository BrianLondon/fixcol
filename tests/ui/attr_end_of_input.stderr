error: fixcol-derive error: Expected value found end of input.
 --> tests/ui/attr_end_of_input.rs:5:31
  |
5 |     #[fixcol(width = 5, align =)]
  |                               ^
